cluster/examples/kubernetes/ceph/monitoring/prometheus_rule_config.json

```diff
@@ -0,0 +1,3 @@
+{
+    "14" : "prometheus-ceph-rules.yaml"
```

Comment:
I'd suggest the following change to simplify:
- Remove this json file
- Name the prometheus rules file with the version embedded in it. For example: prometheus-ceph-rules-14.yaml. When octopus comes out we will have prometheus-ceph-rules-15.yaml. 
We have very tight control about what is in the image, so we should be able to use the predictable filenames instead of a separate configuration file, right?

Reply:
Agree.